### ISSUE TITLE
Eating won't work (heroku)

### ISSUE THREAD

**Issue by PortPixel** [huytd/agar.io-clone#442]
Eating won't work properly - it seems you have to wait under a cell for ~10 seconds to be eaten (this may just be certain positions though)

**Comment by PortPixel** [comment]
That what I though (a commit I saw at same point updated the code to use the default player mass as the radius of cells, before this eating didn't work at all). I'll look though the code when I get a computer (I'm on my phone now) and try to fix it

**Comment by PortPixel** [comment]
I don't think it has to be the exact same position, just a very small radius/only a certain distance perhaps?

**Comment by wb9688** [comment]
Yes, that was my commit. It was a really quick fix for making eating work
again. I should really update it to change the radius to the current player
mass.

**Comment by PortPixel** [comment]
Is anyone planning to fix this? I have tried but it didn't seem to work

**Comment by valonymeri** [comment]
Is this fixed yet ?

**Comment by TheCodeCrafter** [comment]
Yes.

**Comment by TheCodeCrafter** [comment]
The reason that some things are not eaten properly, is mainly because the radius calculations are off, and you have to be larger than one would originally think to consume another cell. This can be easily fixed by changing the calculations, however, in essence, yes it has been fixed.

**Comment by matthall1998** [comment]
How does one change these calculations? Be cool to put it in the config file to make it an easy change.

**Comment by baiplr** [comment]
I remember having this problem a little while ago. I believe that you need to go into the src/server/server.js file and you need to find the collision lines and manually increase the radius of the collision boundary.

**Comment by Ayelis** [comment]
Is this also a duplicate of #413? And is #432 related somehow?

**Comment by TheCodeCrafter** [comment]
> Is this also a duplicate of #413? And is #432 related somehow?

Not exactly, #413 is more about an actual error (if you scroll down a bit), and #423 is a fix for that error. This is more about the default configurations being a little messed up, and it is really based on preference. I think this was originally a problem, then it just became a configurable attribute in server.js

**Comment by Ayelis** [comment]
Ah, but if 413 was truly fixed by 432, wouldn't they both be closed by now? :o ;)

**Comment by TheCodeCrafter** [comment]
Supposedly, it is fixed :D
